doc/api/buffer.md

```diff
@@ -52,15 +52,15 @@ In versions of Node.js prior to 6.0.0, `Buffer` instances were created using the
 differently based on what arguments are provided:
 
 * Passing a number as the first argument to `Buffer()` (e.g. `new Buffer(10)`)
-  allocates a new `Buffer` object of the specified size. Prior to Node.js 8.0.0,
+  allocates a new `Buffer` object of the specified size. Prior to Node.js v8.0.0,
```

Comment:
In prose, we've dropped the `v` to avoid confusion with V8. Phonetically, it's just "node 8" and not "node v8" and our writing should reflect that. We'll never get it out of the version string returned by `--version` and `-v`, but we do what we can.

```suggestion
  allocates a new `Buffer` object of the specified size. Prior to Node.js 8.0.0,
```